File: MixedFractions.java
/**
 * Ladder by Martin Björling, 
 * martinbjorling@gmail.com, 0737565044
 *
 * @author   Martin Björling
 * @version  1.0
 * @since    2015-11-04
 */

import java.lang.Math;
import java.util.Scanner;
import java.util.ArrayList;

public class MixedFractions {
	
	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);

		ArrayList<String> answers = new ArrayList<String>();

		while (true) {
			String inputString = sc.nextLine();

			if (inputString.equals("0 0")) break;

			String[] inputArr = inputString.split(" ");

			int numerator = Integer.parseInt(inputArr[0]);
			int denominator = Integer.parseInt(inputArr[1]);

			int wholeNumber = numerator / denominator;
			int newNumerator = numerator % denominator;

			answers.add(Integer.toString(wholeNumber) + " " + Integer.toString(newNumerator) + " / " + Integer.toString(denominator));
		}
		
		for (int j = 0; j < answers.size(); j++) {
			System.out.println(answers.get(j));
		}

		
	}
}

File: Mixed Fractions/mixedfractions.php
<?php
/**
 * Problem: Mixed Fractions
 * https://open.kattis.com/problems/mixedfractions
 *
 * @author   Martin Björling
 * @email    martinbjorling@gmail.com
 * @version  1.0
 * @since    2015-11-24
 */

$answers = [];

while (true) {
	$inputString = stream_get_line(STDIN, 1024, PHP_EOL);

	if ($inputString == "0 0") break;

	$inputArr = explode(" ", $inputString);

	$numerator = $inputArr[0];
	$denominator = $inputArr[1];

	$wholeNumber = floor($numerator / $denominator);
	$newNumerator = $numerator % $denominator;
	

    $answers[] = $wholeNumber." ".$newNumerator." / ".$denominator."\n";

    
}

foreach ($answers as $value) {
	fwrite(STDOUT, $value);
}


?>